systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -223,22 +228,79 @@ void testAutoRenewCaCertsTriggeredByAnno() throws Exception {
 
         waitForClusterAvailabilityTls(userName, NAMESPACE, CLUSTER_NAME);
 
-        // Finally check a new client (signed by new client key) can consume
+        // Check a new client (signed by new client key) can consume
         String bobUserName = "bob";
         testMethodResources().tlsUser(CLUSTER_NAME, bobUserName).done();
-        waitFor("", Constants.GLOBAL_POLL_INTERVAL, Constants.TIMEOUT_FOR_GET_SECRETS, () -> {
-            return kubeClient().getSecret(bobUserName) != null;
-        },
+        StUtils.waitForSecretReady(bobUserName);
```

Comment:
My suggestion was not precise - lines 235-240 should be removed with this change, now it case build failures. Sorry about that.